service/matching/matchingEngine.go

```diff
@@ -779,6 +779,14 @@ func (e *matchingEngineImpl) createPollActivityTaskQueueResponse(
 
 	serializedToken, _ := e.tokenSerializer.Serialize(taskToken)
 
+	// ScheduleToCloseTimeout can be 0, meaning no timeout
```

Comment:
nit: fix formatting.  And we want to add a comment to deprecate this once all client are updated to the fix which allows handling of infinite timeout.